### ISSUE TITLE
Implement new thermostat / energy API

### ISSUE THREAD

**Issue by cbornet** [cbornet/netatmo-swagger-decl#22]
The current one has been deprecated

**Comment by clinique** [comment]
Yes, I've seen that. I started to adapt the yaml file accordingly but many things has been deprecated and it seems that the way it evolves tends to hide more direct module / device access in favor on a more abstract approach.
